eclair-core/src/main/resources/reference.conf

```diff
@@ -191,63 +191,112 @@ eclair {
-      ratio-cltv = 0.05             // when computing the weight for a channel, consider its CLTV delta in this proportion
-      ratio-channel-age = 0.4       // when computing the weight for a channel, consider its AGE in this proportion
-      ratio-channel-capacity = 0.55 // when computing the weight for a channel, consider its CAPACITY in this proportion
-
-      // Virtual fee for additional hops
-      // Corresponds to how much you are willing to pay to get one less hop in the payment path
-      hop-cost-base-msat = 500
-      hop-cost-millionths = 200
-
-      // Examples:
-      // To optimize for fees only:
-      // ratio-base = 1
-      // ratio-cltv = 0
-      // ratio-channel-age = 0
-      // ratio-channel-capacity = 0
-      // hop-cost-base-msat = 0
-      // hop-cost-millionths = 0
-      //
-      // To optimize for shorter paths:
-      // ratio-base = 1
-      // ratio-cltv = 0
-      // ratio-channel-age = 0
-      // ratio-channel-capacity = 0
-      // hop-cost-base-msat = 10000 // High hop cost penalizes strongly longer paths
-      // hop-cost-millionths = 10000
-      //
-      // To optimize for successful payments:
-      // ratio-base = 0
-      // ratio-cltv = 0
-      // ratio-channel-age = 0.5 // Old channels should have less risk of failures
-      // ratio-channel-capacity = 0.5 // High capacity channels are more likely to have enough liquidity to relay our payment
-      // hop-cost-base-msat = 1000 // Less hops means less chances of failures
-      // hop-cost-millionths = 1000
-      //
-      // To optimize for fast payments:
-      // ratio-base = 0.2
-      // ratio-cltv = 0.5 // In case of failure we want our funds back as fast as possible
-      // ratio-channel-age = 0.3 // Older channels are more likely to run smoothly
-      // ratio-channel-capacity = 0
-      // hop-cost-base-msat = 10000 // Shorter paths should be faster
-      // hop-cost-millionths = 10000
-
-      mpp {
-        min-amount-satoshis = 15000 // minimum amount sent via partial HTLCs
-        max-parts = 5 // maximum number of HTLCs sent per payment: increasing this value will impact performance
+      default {
+        randomize-route-selection = true // when computing a route for a payment we randomize the final selection
+
+        boundaries {
+          max-route-length = 6         // max route length for the 'first pass', if none is found then a second pass is made with no limit
+          max-cltv = 1008              // max acceptable cltv expiry for the payment (1008 ~ 1 week)
+          // the route must have a fee that's below max-fee-flat-sat OR below max-fee-proportional-percent / 100 * total-amount
+          max-fee-flat-sat = 21
+          max-fee-proportional-percent = 3 // that's 3%
+        }
+
+        // channel 'weight' is computed with the following formula: (channelFee + hop-cost) * (ratio-base + cltvDelta * ratio-cltv + channelAge * ratio-channel-age + channelCapacity * ratio-channel-capacity)
```

Comment:
Noob question. Why isn't the formula:

`(channelFee + hop-cost) + cltvDelta * ratio-cltv + channelAge * ratio-channel-age + channelCapacity * ratio-channel-capacity`

Just summing the weights.

Reply:
It could be, that probably makes more sense actually.

Reply:
I think the reason we currently do that is because we want the fee to have a scaling impact even when ratios for other things are high. We internally always bias towards the fee as the most important parameter (and I think it's ok).

We could have an experiment where we change that formula though!

Reply:
> We could have an experiment where we change that formula though!

Right, using an additional parameter to select the weight calculation mode. I guess let's revisit this later and not delay this PR further then